src/base-widget.ts

```diff
@@ -20,12 +21,16 @@ export abstract class BaseWidget implements OnInit, OnDestroy {
   public state?: object = {};
   public cx: ReturnType<typeof bem>;
   public abstract instantSearchParent: NgAisInstantSearch;
```

Comment:
The term "parent" in `instantSearchParent` becomes confusing now that we have `indexParent`. Perhaps either rename to `instantSearch` or `instantSearchInstance` (term used in IS.js)?